### PR TITLE
News link

### PR DESCRIPTION
**Pull request by davidrapson**
image

### CODE REVIEW

File: views/components/news.njk [hunk 1]
```diff
@@ -13,3 +13,28 @@
...
     </article>
 {% endmacro %}
+
+{% macro latestNews(articles) %}
+    <div class="latest-news">
+        <div class="latest-news__header">
+            <h3 class="t1 t--underline a--text accent--blue">
+                <a class="u-link-minimal" href="{{ buildUrl("uk-wide/news-and-events") }}">
+                    {{ __('global.misc.latestNews') }}</h3>
+                </a>
```
Comment (mattandrews) [inline views/components/news.njk]: Missing a closing `</h3>` tag here?

File: assets/sass/components/news.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/* =========================================================================
+   News
+   ========================================================================= */
+
+// .article-teaser {}
```
Comment (mattandrews) [inline assets/sass/components/news.scss]: Could delete this?

Comment (davidrapson) [reply assets/sass/components/news.scss]: Could, but I kind of like an explicit comment saying what the parent class  is if it doesn't have any styles attached.

File: views/components/news.njk (hunk 1 above)
Comment (davidrapson) [reply views/components/news.njk]: Good catch

Comment (davidrapson) [reply views/components/news.njk]: (This has been corrected, but the comment is on the `</a>` so the comment isn't dismissed.

**Pull request merged**